autoload/vim_dadbod_completion/schemas.vim

```diff
@@ -23,6 +23,17 @@ let s:postgres = {
       \ 'count_parser': function('s:count_parser', [1])
       \ }
 
+let s:oracle_args = "echo \"SET linesize 4000;\nSET pagesize 4000;\n%s\" | "
```

Comment:
I had to create separate variables, since Oracle's system catalog structure is quite different from MySQL and Postgres.